tools/WLED_ESP32_NO_OTA.csv

```diff
@@ -0,0 +1,5 @@
+# Name,   Type, SubType, Offset,  Size, Flags
```

Comment:
Is the code size >1.5MB?

Reply:
> Is the code size >1.5MB?

Yes, 1.8MB+ with the compile options in the usermod readme

Reply:
No OTA will be a hard sell.